templates/data/kyma-integration-data.yaml

```diff
@@ -540,6 +543,9 @@ templates:
                     - "vm_job_labels_template"
               - jobConfig:
                   name: "pre-main-kyma-prod-istio-integration-k3d"
+                  annotations:
+                    owner: goat
+                    description: runs istio component tests
```

Comment:
We shouldn't have the same description as for `pre-main-kyma-eval-istio-integration-k3d`, since this job at least uses a different profile.